flogging/flogging.py

```diff
@@ -219,6 +219,8 @@ def emit(self, record: logging.LogRecord):
             "thread": reduce_thread_id(record.thread),
             "name": record.name,
         }
+        if "RANK" in os.environ:
```

Comment:
Save one dict lookup:
```suggestion
        try:
            rank = os.environ["RANK"]
        except KeyError:
            pass
        else:
            obj["RANK"] = rank
```